Source/Engine/Syntax/TextRange.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Nezaboodka.Nevod
+{
+    public struct TextRange
+    {
+        public int Start { get; set; }
+        public int End { get; set; }
```

Comment:
I think Start and End can be readonly properties without setters (unless I am missing places where setters are used).